service/upload/upload.go

```diff
@@ -1,12 +1,15 @@
 package uploadService
 
 import (
+	"regexp"
 	"strings"
 
 	"github.com/NyaaPantsu/nyaa/config"
 	"github.com/NyaaPantsu/nyaa/model"
 )
 
+const trackerRegex = `^(http[s]*|udp)://[a-z0-9\.:\-/?]+$`
```

Comment:
tracker urls can contain `&`, `=`, `[` and `]` too
why are you even validating this?